### PR TITLE
e2e ginkgo resource

### PR DESCRIPTION
**Pull request by wwwtyro**
~~Looking into some failing tests - don't merge yet.~~

### CODE REVIEW

File: cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import os
-import platform
```
Comment (wwwtyro) [inline cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py]: flake8

File: cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py [hunk 2]
```diff
@@ -49,6 +48,7 @@ def messaging():
 
     hookenv.status_set('active', 'Ready to test.')
 
+
```
Comment (wwwtyro) [inline cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py]: flake8

File: cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py [hunk 3]
```diff
@@ -78,23 +78,22 @@ def install_kubernetes_e2e():
                            'Incomplete {} resource.'.format(resource))
         return
 
-    hookenv.status_set('maintenance', 'Unpacking {} resource.'.format(resource))
+    hookenv.status_set('maintenance', 'Unpacking {} resource.'.format(resource))  # noqa
```
Comment (wwwtyro) [inline cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py]: flake8

File: cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py [hunk 4]
```diff
@@ -78,23 +78,22 @@ def install_kubernetes_e2e():
...
-    services = ['e2e.test', 'kubectl']
+    services = ['e2e.test', 'ginkgo', 'kubectl']
 
     for service in services:
         unpacked = '{}/{}'.format(unpack_path, service)
         app_path = '/usr/local/bin/{}'.format(service)
         install = ['install', '-v', unpacked, app_path]
         call(install)
 
-
```
Comment (wwwtyro) [inline cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py]: flake8

**Comment by Cynerva** [comment]
Looks good to me.

**Comment by wwwtyro** [comment]
@mbruzek @chuckbutler The issues I was seeing were unrelated to these changes. This tests out & is ready for a low-priority review at your leisure.

File: cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py (hunk 3 above)
Comment (chuckbutler) [inline cluster/juju/layers/kubernetes-e2e/reactive/kubernetes_e2e.py]: can we solve this another way than using #noqa?

**Pull request merged**